plugin/src/main/java/com/fourlastor/pickle/PicklePlugin.kt

```diff
@@ -151,7 +183,7 @@ class PicklePlugin : Plugin<Project> {
      *
      * Remove this when we want to only support Android Gradle Plugin 3.3 and above.
      */
-    private fun setupDependency(task: GenerateTask, variant: TestVariant) {
+    private fun setupDependency(task: GenerateTask, variant: BaseVariant) {
```

Comment:
Since this is called for both tasks, accessing `mergeAssets` causes

> Caused by: kotlin.UninitializedPropertyAccessException: lateinit property mergeAssetsTask has not been initialized


The dependency should be different for unit tests.

Reply:
You're very right :man_facepalming: my bad, forgot about this one